Source/WebCore/platform/Decimal.cpp

```diff
@@ -750,19 +749,6 @@ Decimal Decimal::fromString(StringView str)
             return nan();
 
         case StateDot:
-            if (isASCIIDigit(ch)) {
-                if (numberOfDigits < Precision) {
-                    ++numberOfDigits;
-                    ++numberOfDigitsAfterDot;
-                    accumulator *= 10;
-                    accumulator += ch - '0';
-                }
-                state = StateDotDigit;
-                break;
-            }
-            // FIXME: <http://webkit.org/b/127667> Decimal::fromString's EBNF documentation does not match implementation
```

Comment:
The issue still remains, right? If so we shouldn't remove this FIXME

Reply:
I think I copied my local changes, where I usually don't add comments etc., let me add it back.

Reply:
Done in next commit.